source/less/blocks/card.less

```
@@ -1,27 +1,92 @@
 .card {
-  padding: 20px @padding-site;
+
+  padding: 0 @mobile-padding;
   padding-bottom: 25px;
-  border-top: 1px solid @special-grey-2;
+  border-bottom: 1px solid @catalog-border;
+  margin: 0 auto;
+  width: @mobile-width - @mobile-padding * 2;
+
+
+  @media (min-width: @tablet-width) {
+    width: 324px;
+    padding: 0 0;
+    padding-top: 161px;
+    padding-bottom: 38px;
```

Так лучше не писать, это дублирование
```suggestion
    padding: 161px 0 38px;
```

лучше везде исправить